source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,57 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes of data in MongoDB, such as changes to a
+collection, database, or deployment by opening a :manual:`Change
+Stream</changeStreams/>`. A ``Change Stream`` allows applications to
+watch for changes to data and react to those changes. You can open a
+``Change Stream`` by calling the :node-api:`watch()
+</Collection.html#watch>` method on a collection, database, or
+deployment. The ``watch()`` method takes an optional pipeline as the
+first parameter that is an array of stages to filter the :manual:`change
+events </reference/change-events/>` output. 
+
+The ``watch()`` method can take in additional options as the second
+parameter. Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set ``fullDocument`` to
+``'updateLookup'`` to receive an event that contains the changes to the
+document as well as the entire altered document.
+
+Process the Change Stream Events
+--------------------------------
+
+The ``watch()`` command returns a ``Change Stream`` instance. Use the
+`EventEmitter.on()
+<https://nodejs.org/api/events.html#events_emitter_on_eventname_listener>`_.
+method on the ``Change Stream`` instance to add a listener function. Call
+``changeStream.on()`` with the string ``'change'`` as the
+first parameter and a :mdn:`callback function
+<Glossary/Callback_function>` as the second parameter. The callback will
+trigger when a ``change event`` is emitted, providing the next available
+document. You can use the callback to specify logic to process the
+document when the event is received. 
+
+To free up resources once you are done processing the change events,
+call the :node-api:`close() </ChangeStream.html#close>` method to close
+the change stream instance and stop watching for changes.
```

Comment:
Nitpick: This sentence contains two actions ("call close()" and "to close and stop") rather than a single one which makes it less easy to read. Either split this into two sentences or omit one of the actions. E.g.:
1) Call the close() method on the change stream instance to stop processing change events. This method closes the change stream and frees resources.
2) To stop watching for changes, call close() on the change stream instance.

Reply:
Thanks for the suggested sentences, I've replaced my original with your first sentence which captures what I intended well and sounds clean.